swift/integration-tests/osx-only/canonical-case/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+SDK="-sdk $(xcrun -show-sdk-path)"
+FRONTEND="$(xcrun -find swift-frontend)"
+
+# This test case will only work on a case-insensitive FS (which is the default)
```

Comment:
You can totally have a case-sensitive FS on macOS, but I expect this to be "I know exactly what I'm doing." The comment here is simply to help someone hitting this issue to understand where it comes, in case this ever happens.